src/context.rs

```diff
@@ -14,4 +15,23 @@ pub struct Context {
     pub gtfs_rt: Arc<Mutex<Option<GtfsRT>>>,
     pub gtfs: gtfs_structures::Gtfs,
     pub gtfs_rt_provider_url: String,
+    pub lines_of_stops: HashMap<String, HashSet<String>>,
```

Comment:
I would have prefered something more encapsulated, this context is starting to have some quite different stuff, but I do agree on the fact that we'll be able to do it later on